torch/utils/collect_env.py

```diff
@@ -244,6 +253,17 @@ def get_os(run_lambda):
     # Unknown platform
     return platform
 
+def get_kernel_version():
+    import platform
+    if get_platform() == 'linux':
+        return '{} {}'.format(platform.release(), platform.version())
+    else:
+        return platform.release()
+
+def get_libc_version(run_lambda):
```

Comment:
there's also `platform.libc_ver()` if that works better?

Reply:
echo on the comment https://github.com/pytorch/pytorch/pull/58485#discussion_r634573136.